src/hooks/usePairLiquidityMiningCampaigns.ts

```diff
@@ -0,0 +1,127 @@
+import { useCallback, useMemo } from 'react'
+import { useQuery } from '@apollo/client'
+import { BigintIsh, Pair } from '@swapr/sdk'
+import { SubgraphLiquidityMiningCampaign } from '../apollo'
+
+import { useNativeCurrency } from './useNativeCurrency'
+import { useActiveWeb3React } from '.'
+import { getLowerTimeLimit, getTokenAmount, toLiquidityMiningCampaign } from '../utils/liquidityMining'
+import { useAllTokensFromActiveListsOnCurrentChain } from '../state/lists/hooks'
+import { useKpiTokens } from './useKpiTokens'
+import { PairsFilterType } from '../components/Pool/ListFilter'
+import { useSWPRToken } from './swpr/useSWPRToken'
+import { REGULAR_CAMPAIGN, sortActiveCampaings, sortExpiredCampaigns } from './useAllLiquidityMiningCampaigns'
```

Comment:
`sortActiveCampaings, sortExpiredCampaigns` could be in `utils/liquidityMining`

Reply:
Typo in `sortActiveCampaings`

Reply:
Thanks. Will fix.